blinkit/page-1.html

```diff
@@ -0,0 +1,328 @@
+<!DOCTYPE html>
```

Comment:
Points to keep in mind while raise a pull request

1. The branch name should be descriptive -- development-1 is not a proper name for the branch
2. Description should give details about the changes in the PR. Always include screenshots / videos (whatever applicable) showcasing the behaviour added in the PR